enhancement(metrics): allow disabling via config option (#4165)
hedgedoc/hedgedoc: public/docs/release-notes.md

- Add dedicated healthcheck endpoint at /_health that is less resource intensive than /status.
- Compatibility with Node.js 18 and later
- Add support for the arm64 architecture in the docker image

> - Allow disabling the exposure of the /status and /metrics endpoints via the `CMD_ENABLE_STATS_API` environment variable or the config option `enableStatsApi`. It is enabled by default to not break existing setups.

Reviewer: ```suggestion
- Add a config option to disable the `/status` and `/metrics` endpoints
```
